WEB_SERVICE/WEB/_HIGH/FIELD_DUPLICATION/meta.json

```diff
@@ -0,0 +1,37 @@
+{
+    "risk_rating": "high",
+    "title": "Field Duplication in GraphQL API",
+    "short_description": "Exploiting a GraphQL server by duplicating fields, potentially leading to Denial of Service attacks.",
+    "references": {
+      "imperva": "https://www.imperva.com/blog/graphql-vulnerabilities-common-attacks/",
+      "escape": "https://docs.escape.tech/testing/vulnerabilities/resource_limitation/graphql_field_duplication",
+      "Sylhare": "https://sylhare.github.io/2024/04/19/Graphql-vulnerabilities.html"
+    },
+    "privacy_issue": false,
+    "security_issue": true,
+    "categories": {
+      "CWE_TOP_25": [
+        "CWE_400"
+      ],
+      "PCI_STANDARDS": [
+        "REQ_6_2",
+        "REQ_6_4",
+        "REQ_11_3"
+      ],
+      "OWASP_MASVS_L2": [
+        "MSTG_PLATFORM_2"
```

Comment:
I think other MASVS are missing, L2 seems to high for this?

Reply:
i did it based on other KBs that are previously merged